NOTICE-binary

```diff
@@ -123,3 +123,15 @@ This software includes projects with other licenses -- see `doc/LICENSE.md`.
 
 This product includes software developed by Google
  Snappy: http://code.google.com/p/snappy/ (New BSD License)
+
+MyBatis
+Copyright 2010-2023
+
+This product includes software developed by
+The MyBatis Team (https://www.mybatis.org/).
+
+iBATIS
+   This product includes software developed by
+   The Apache Software Foundation (https://www.apache.org/).
+
+   Copyright 2010 The Apache Software Foundation
```

Comment:
the reminding content is required too, because those part is included in mybatis jar and shipped by our binary tgz

Reply:
For the iBATIS part, we can simplify it to

```
iBATIS
   Copyright 2010 The Apache Software Foundation
```